stacks-signer/src/client/stacks_client.rs

```diff
@@ -757,16 +899,46 @@ mod tests {
 
     #[ignore]
     #[test]
-    fn transaction_contract_call_should_succeed() {
+    #[serial]
+    fn build_vote_for_aggregate_public_key_should_succeed() {
```

Comment:
I'd love to see a test that actually evaluates the transaction payload in a ClarityVM. I think if you could either use or tweak `signers_voting_tests.rs` it shouldn't be too hard

Reply:
Will have to do in a subsequent PR.